tools/c7n_gcp/c7n_gcp/regions.json

```diff
@@ -41,5 +41,8 @@
   "us-west2",
   "us-west3",
   "us-west4",
-  "us-west8"
+  "us-west8",
```

Comment:
service specific augments wrt to annotating is probably a better way to address. this seems to break many things to try and make a specific service work for additional regions.

also the data dictionary update is automated via makefile target which will override these manual entries.

Reply:
Okey, i rollbacked this list. 
Please look into code: i kept `artifact-repository` a child resource but changed internal behaviour in favor of using a custom request to get regions instead of using `gcp.region` as source. `gcp.region` behaviour seems somewhat strange to me. I don't get why we need to check `self.config.region != 'us-east-1'` because as far as i debugged `config.region` is always equal to an empty string. But changing that logic makes some tests fail so i'm not sure.

I can rollback all changes in `gcp.region` whatsoever. The main concert here is about not working `gcp.artifact-repository`